src/screens/FeedScreen/Header.js

```diff
@@ -220,6 +221,7 @@ const _renderProfileNormal = ({
               size={50}
               width={3}
               withKarma
+              onImageContainerClick={navigateToProfile}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding an `onImageContainerClick` event handler is a good way to make the image interactive. Ensure that the `navigateToProfile` function is defined and correctly handles navigation.
```diff
+              onImageContainerClick={navigateToProfile}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->